Source/Csla.Shared/Reflection/ServiceProviderMethodCaller.cs

```diff
@@ -214,7 +238,12 @@ public static ServiceProviderMethodInfo FindDataPortalMethod<T>(Type targetType,
               }
               else
               {
+#if NET5_0_OR_GREATER
```

Comment:
Isn't this #if block invoking the same line of code either way?

Reply:
Yep, value getter places have null-checks. Missed on refactoring.

For previous comment, resolved it without comment:

I have no complains) You can name it whatever you like. I'm not sure that it has something with reflection, active ```AssemblyLoadContext``` just provides "closest" context for types caching/searching. Contexts are linked together via "child" -> "parent" chain, up to main, "Default", application context (like legacy ```AppDomain```). So if type was registred in "Default" application context, but was accessed in some "dynamic" context, it will be looked up to "Default" context on this contexts chain, something like that.